Cargo.toml

```diff
@@ -124,6 +124,9 @@ debug = 0
 # for the extra time of optimizing it for a clean build of matrix-sdk-ffi.
 quote = { opt-level = 2 }
 sha2 = { opt-level = 2 }
+# faster runs for insta.rs snapshot testing
```

Comment:
I think it is the right place to put it? but maybe we need also in other conf? 
It is recommened to do that in https://insta.rs/docs/quickstart/

Reply:
Yes.